### PR TITLE
feat(aquavm-air-cli): `--near` execution mode [fixes VM-322]

### PR DESCRIPTION
**Pull request by monoid**
The NEAR execution mode that executes AIR NEAR smart contract, measuring
its gas consumption.

### CODE REVIEW

File: tools/cli/air/src/trace/run/near.rs [hunk 1]
```diff
@@ -0,0 +1,125 @@
+/*
```
Comment (raftedproc) [inline tools/cli/air/src/trace/run/near.rs:1]: I am curious if this module should be behind a feature flag near ?

Comment (monoid) [reply tools/cli/air/src/trace/run/near.rs:1]: It is feature-flagged in the parent module.

File: tools/cli/air/src/trace/run.rs [hunk 2]
```diff
@@ -155,25 +217,40 @@ pub(crate) fn run(args: Args) -> anyhow::Result<()> {
...
+            .context("Failed to instantiate NEAR AVM"),
     }
 }
 
 #[cfg(not(feature = "wasm"))]
 fn get_runner(
-    _native: bool,
+    mode: Option<Mode>,
     _air_interpreter_wasm_path: &Path,
+    _air_contract_wasm_path: &Path,
```
[the remark this answers is not in the document]
Comment (monoid) [reply tools/cli/air/src/trace/run.rs:242]: Exactly. `_` prefix doesn't mean "unused", it means "I'm fine if it is unused".

[the remark this answers is not in the document]
Comment (raftedproc) [reply tools/cli/air/src/trace/run.rs:242]: The variable name blows up my mind TBH. Did I get it right the variable points to wasm module that contains AIR wrapped in NEAR?

[the remark this answers is not in the document]
Comment (monoid) [reply tools/cli/air/src/trace/run.rs:242]: To a WASM module which is a NEAR contract executing AIR (see `tools/wasm/air-near-contract`).

[the remark this answers is not in the document]
Comment (raftedproc) [reply tools/cli/air/src/trace/run.rs:242]: I would suggest near_contract_running_air_wasm_path. It is a bit lengthy but gives additional semantics about who does what.

File: tools/wasm/air-near-contract/Cargo.toml [hunk 3]
```diff
@@ -1,6 +1,6 @@
 [package]
 name = "air-near-contract"
-version = "0.43.1"
+version = "0.1.0"
```
Comment (monoid) [inline tools/wasm/air-near-contract/Cargo.toml:3]: It's version should match `air`'s version, as it is just a thin wrapper.

**Comment by monoid** [comment]
utACK

File: tools/wasm/air-near-contract/Cargo.toml (hunk 3 above)
Comment (mikevoronov) [reply tools/wasm/air-near-contract/Cargo.toml]: makes sense

**Review by mikevoronov — APPROVED** [review]
utACK

**Pull request merged**